In [4]:
import numpy as np

#### 1.Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

## $\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$  (1)

$2Ф (t_{\lambda}) = \lambda$

$\lambda = 0.95$

$2Ф (t_{\lambda}) = 0.95$

$Ф (t_{\lambda}) = 0.475$

Найдем по таблице значений функции Лапласа значение функции 0.475 и соответствующее ему значение $ t_{\lambda}$

$t_{\lambda} = 1.96$

Подставим найденное значение в (1) и найдем $\delta$:

In [5]:
delta = 1.96 * 16 / np.sqrt(256)
delta

1.96

Найдем доверительный интервал: 

In [6]:
80 - delta

78.04

In [7]:
80 + delta

81.96

Ответ: доверительный интервал $(78,04 ; 81,96)$

#### 2.В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1. Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [8]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

Найдем среднее значение:

In [9]:
Mean = np.mean(a)
Mean

6.590000000000001

In [10]:
Std = np.std(a)
Std

0.42766809560686203

## $\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$ 

Т.к.  надежность соответствует надежности в первой задаче, то возьмем значение $t_{\lambda} =1.96$ из нее и поставим все данные в формулу:

In [11]:
delta = 1.96 * Std / np.sqrt(np.count_nonzero(a))
delta

0.2650714318820495

Найдем доверительный интервал:

In [12]:
Mean - delta

6.324928568117951

In [13]:
Mean + delta

6.855071431882051

Ответ: доверительный интервал $(6.324928568117951 ; 6.855071431882051)$

#### 3.Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

Нулевая гипотеза:

H0 : M = 17 мм

Альтернативная гипотеза:

H1: M > 17 мм

In [36]:
U_nabl = ((17.5 - 17) / np.sqrt(4)) * np.sqrt(100)
U_nabl

2.5

Найдем критическую точку для односторнней критической области:

In [18]:
alpha = 0.05

In [20]:
Ф = (1 -2*alpha)/2
Ф

0.45

По таблице находим $\theta$

In [ ]:
u_kr = 1.645

Наблюдаемое значение (U_nabl) больше критического значения (u_kr)

Ответ: Гипотеза о том, что средний диаметр 17 мм не верна. Принимаем альтернативную гипотезу.

#### 4.Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:202, 203, 199, 197, 195, 201, 200, 204, 194, 190.Известно, что их веса распределены нормально.Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

Нулевая гипотеза:

H0: M = 200 г

Альтернативная гипотеза:

H1: M > 200 г

In [26]:
a = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [27]:
Mean = np.mean(a)
Mean

198.5

In [38]:
Std = np.std(a)
Std

4.224926034855522

In [39]:
U_nabl = ((200 - Mean)/Std) * np.sqrt(np.count_nonzero(a))
U_nabl

1.1227217828476797

u_kr = 1.65, т.к. доверительная вероятность такая же,как в предыдущей задаче.

Наблюдаемое значение (U_nabl) меньше, чем критическое значение(u_kr)

Ответ: принимаем H0.